.github/workflows/ci-config-functions.yml

```diff
@@ -33,17 +33,13 @@ jobs:
           path: |
             ~/.cache/pip
             ~/.cache/pypoetry
-          key: pip-${{ hashFiles('.virtualenv.dev-requirements.txt') }}-${{ hashFiles('.virtualenv.requirements.txt') }}
-          restore-keys: |
-            -pip-
```

Comment:
Why did you remove this? It ensures that the action restores the correct cache

Reply:
These files don't exist anymore since we moved to poetry.